packages/big-design/src/components/Input/Input.tsx

```diff
@@ -161,13 +121,62 @@ class StyleableInput extends React.PureComponent<InputProps & PrivateProps, Inpu
         <Chip key={key} label={chip} marginBottom="none" marginTop="none" />
       ),
     );
-  }
-}
+  }, [chips, onChipDelete]);
 
-const InputWithForwardedRef = React.forwardRef<HTMLInputElement, InputProps>(({ className, style, ...props }, ref) => (
-  <StyleableInput {...props} forwardedRef={ref} />
-));
+  const isErrorValid = (err: Props['error']) => {
```

Comment:
Can we rename this function? I would expect this to return a `boolean` by its name.